Client/Frontend/Browser/Playlist/Controllers/PlaylistListViewController+TableViewDelegate.swift

```diff
@@ -173,9 +178,107 @@ extension PlaylistListViewController: UITableViewDelegate {
         return UISwipeActionsConfiguration(actions: [deleteAction, shareAction, cacheAction])
     }
     
+    func tableView(_ tableView: UITableView, contextMenuConfigurationForRowAt indexPath: IndexPath, point: CGPoint) -> UIContextMenuConfiguration? {
+        
+        if indexPath.row < 0 || indexPath.row >= PlaylistManager.shared.numberOfAssets {
+            return nil
+        }
```

Comment:
No not necessary anymore. I forgot that I modified itemAtIndex to do bounds checking because we used to have huge crashes in CoreData for `Bookmarks` without it. I ended up doing the same for playlist, but just never removed from all the places it was called in.